Library/Formula/qpm.rb

```diff
@@ -0,0 +1,19 @@
+class Qpm < Formula
+  desc "Package manager for Qt applications"
+  homepage "https://www.qpm.io"
+  url "https://github.com/Cutehacks/qpm.git",
+      :tag => "v0.10.0",
+      :revision => "b0dfc0f26e009ff778cc25ebb8b166fa460ea998"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    system "go", "install", "qpm.io/qpm"
```

Comment:
We prefer to vendor submodules in ourselves using `go_resource`s and then use `go build` instead of `go install`.

Seems to be pulling a lot in:

```
Submodule 'src/github.com/golang/protobuf' (https://github.com/golang/protobuf.git) registered for path 'src/github.com/golang/protobuf'
Submodule 'src/github.com/howeyc/gopass' (https://github.com/howeyc/gopass) registered for path 'src/github.com/howeyc/gopass'
Submodule 'src/golang.org/x/crypto' (https://go.googlesource.com/crypto) registered for path 'src/golang.org/x/crypto'
Submodule 'src/golang.org/x/net' (https://go.googlesource.com/net) registered for path 'src/golang.org/x/net'
Submodule 'src/google.golang.org/grpc' (https://github.com/grpc/grpc-go) registered for path 'src/google.golang.org/grpc'
Cloning into 'src/github.com/golang/protobuf'...
```